lib/controller.js

```diff
@@ -705,39 +710,152 @@ controller.disableAccessPoint = function(opts) {
   });
 };
 
+// clear terminal windows and linux
+controller.clear = function() {
+  return new Promise(function(resolve) {
+    // clears terminal
+    if (!global.IS_TEST_ENV) {
+      process.stdout.write('\u001b[2J\u001b[0;0H');
```

Comment:
http://stackoverflow.com/questions/22488850/how-can-i-clear-a-windows-7-console-screen-between-function-calls-in-node-js/22494170#22494170